todo.md

```diff
@@ -32,32 +32,29 @@ registration, and by outreach through education.
 
 ### Abstract/Proposal Summary (required) maximum of 250 words
 
-Our proposal has two major components:
-
-* Strengthening code foundations to make Nibabel more useful as an *brain
-  image workbench* and *base library*.
-* Broadening social foundations by building a platform and content for
-  teaching neuroimaging with Nibabel.
-
-We will *strengthen code foundations* by:
-
-* Expanding the API to support image metadata by adding axis and tick labels.
-* Extending support for reading and storing metadata from scanner outputs.
-* Extending output formats to store metadata.
-* Designing a standard API for brain surface data.
-* Adding support for loading, manipulating, applying and storing outputs from
-  image registration.
-
-We will *broaden social foundations* by:
-
-* Creating an open-access online course to teach neuroimaging using Nibabel as
-  brain image workbench, built on existing materials from our neuroimaging teaching in Berkeley.
-* Using the course materials as open, engaging and comprehensive documentation
-  for the library.
-* Recruiting a broader range of developers with targeted scholarships in the
-  form of paid course certificates.
-* Building and documenting the platform for delivering the course so that other
-  fields and libraries can build on this foundation.
+Nurturing and engaging a diverse community around base libraries like Nibabel is challenging.
+We will lower the entry barriers of Nibabel in particular, and more broadly, neuroimaging
+processing and analysis in general.
+Our __Aim 1 will strengthen the *code foundations* of Nibabel as a *brain image workbench*__.
+To achieve that, we will
+adopt a more expressive API supporting more transparent and comprehensive metadata;
+extend the support for reading and storing metadata from scanner outputs;
+design an API for brain surface data;
+and add support for loading, manipulating, applying and storing spatial transformations.
```

Comment:
I''m not a huge fan of "a more expressive API supporting more transparent and comprehensive metadata;", but didn't think of a better phrase. This is mostly plundering for breathing room.

```suggestion
Aim 1 is to strengthen the code foundations of Nibabel.
To achieve this, we will:
develop a more expressive API supporting more transparent and comprehensive metadata;
extend the support for reading and storing metadata from scanner outputs;
design a standard API for brain surface data;
and add support for loading, manipulating, applying and storing spatial transformations.

```